nornir_nautobot/plugins/tasks/dispatcher/__init__.py

```diff
@@ -31,6 +31,7 @@ def dispatcher(task: Task, method: str, logger, obj, *args, **kwargs) -> Result:
     """
     if kwargs.get("default_drivers_mapping"):
         default_drivers_mapping = kwargs["default_drivers_mapping"]
+        del kwargs["default_drivers_mapping"]
```

Comment:
Want to call this out as the strategy to let users add their own dispatchers. @jvanderaa @dgarros